integration-cli/docker_cli_help_test.go

```diff
@@ -15,7 +16,13 @@ func TestMainHelpWidth(t *testing.T) {
 
 	var home string
 	if runtime.GOOS != "windows" {
-		home = os.Getenv("HOME")
+		// Only do this on non-windows systems
+		home = homedir.Get()
+
+		// Don't check when HOME is /
+		if home == "/" {
+			home = ""
```

Comment:
Correct, when home is "" I skip the check.  So, $HOME="/" or being on windows will skip it.